### PR TITLE
Fixes falsewalls being unsmashable

### PR DESCRIPTION
**Pull request by Razharas**
Fixes falsewalls being unsmashable
Fix https://github.com/tgstation/-tg-station/issues/15465

### CODE REVIEW

File: code/game/objects/structures/false_walls.dm [hunk 1]
```diff
@@ -48,6 +48,9 @@
 			update_icon()
 	opening = 0
 
+/obj/structure/falsewall/attack_animal(mob/user)
+	qdel(src)
```
Comment (phil235) [inline code/game/objects/structures/false_walls.dm]: corgi deletes falsewalls by clicking them.

Comment (Razharas) [inline code/game/objects/structures/false_walls.dm]: I hoped its handled unifiedly somewhere so i dont need to touch it

File: code/game/objects/structures/false_walls.dm [hunk 2]
```diff
@@ -48,6 +48,13 @@
 			update_icon()
 	opening = 0
 
+/obj/structure/falsewall/attack_animal(mob/living/simple_animal/user)
+	if(user.environment_smash)
+		user.do_attack_animation(src)
```
Comment (phil235) [inline code/game/objects/structures/false_walls.dm]: `user.changeNext_move(CLICK_CD_MELEE)`

File: code/game/objects/structures/false_walls.dm [hunk 3]
```diff
@@ -48,6 +48,13 @@
...
+		playsound(src.loc, 'sound/weapons/Genhit.ogg', 50, 1)
```
Comment (phil235) [inline code/game/objects/structures/false_walls.dm]: probably more appropriate to use the same sound that wall smashing uses: `playsound(src, 'sound/effects/meteorimpact.ogg', 100, 1)`

**Pull request merged**